public/app/core/internationalization/locales.ts

```diff
@@ -1,146 +1,285 @@
+  // Arabic - Algeria
+  { name: 'العربية (الجزائر)', code: 'ar-DZ' },
+  // Arabic - Kuwait
+  { name: 'العربية (الكويت)', code: 'ar-KW' },
+  // Arabic - Libya
+  { name: 'العربية (ليبيا)', code: 'ar-LY' },
+  // Arabic - Morocco
+  { name: 'العربية (المغرب)', code: 'ar-MA' },
+  // Arabic - Palestine
+  { name: 'العربية (فلسطين)', code: 'ar-PS' },
+  // Arabic - Saudi Arabia
+  { name: 'العربية (السعودية)', code: 'ar-SA' },
+  // Arabic - Tunisia
+  { name: 'العربية (تونس)', code: 'ar-TN' },
+  // Azerbaijani - Azerbaijan
+  { name: 'Azərbaycan dili', code: 'az' },
+  // Belarusian - Belarus
+  { name: 'Беларуская мова', code: 'be-BY' },
+  // Bulgarian - Bulgaria
+  { name: 'Български език', code: 'bg-BG' },
+  // Bambara - Mali
   { name: 'Bamanankan', code: 'bm' },
-  { name: 'Bengali', code: 'bn' }, // TODO translate : বাংলা ???
-  { name: 'Bengali (Bangladesh)', code: 'bn-bd' }, // TODO translate
-  { name: 'Tibetan', code: 'bo' }, // TODO translate
+  // Bengali - Standard
+  { name: 'বাংলা', code: 'bn' },
+  // Bengali - Bangladesh
+  { name: 'বাংলা', code: 'bn-BD' },
+  // Tibetan - Tibet (China) and Bhutan
+  { name: 'བོད་ཡིག', code: 'bo' },
+  // Breton - Brittany (France)
   { name: 'Brezhoneg', code: 'br' },
-  { name: 'Босански', code: 'bs' },
-  { name: 'Catalán', code: 'ca' },
-  { name: 'Čeština', code: 'cs' },
-  { name: 'Cymraeg', code: 'cy' },
-  { name: 'Чӑвашла', code: 'cv' },
-  { name: 'Dansk', code: 'da' },
-  { name: 'Deutsch', code: 'de' },
-  { name: 'Deutsch (Österreich)', code: 'de-at' },
-  { name: 'Deutsch (Schweiz)', code: 'de-ch' },
-  { name: 'ދިވެހި', code: 'dv' },
-  { name: 'Ελληνικά', code: 'el' },
-  { name: 'English (Australia)', code: 'en-au' },
-  { name: 'English (Canada)', code: 'en-ca' },
-  { name: 'English (United Kingdom)', code: 'en-gb' },
-  { name: 'English (Ireland)', code: 'en-ie' },
-  { name: 'English (Israel)', code: 'en-il' },
-  { name: 'English (India)', code: 'en-in' },
-  { name: 'English (New Zealand)', code: 'en-nz' },
-  { name: 'English (Singapore)', code: 'en-sg' },
-  { name: 'English (United States)', code: 'en' },
+  // Bosnian - Bosnia and Herzegovina
+  { name: 'Bosanski jezik', code: 'bs' },
+  // Catalan - Catalonia (Spain)
+  { name: 'Català', code: 'ca-ES' },
+  // Czech - Czech Republic
+  { name: 'Čeština', code: 'cs-CZ' },
+  // Welsh - Wales (United Kingdom)
+  { name: 'Cymraeg', code: 'cy-GB' },
+  // Chuvash - Chuvashia (Russia)
+  { name: 'Чӑвашла', code: 'cv-RU' },
+  // Danish - Denmark
+  { name: 'Dansk', code: 'da-DK' },
+  // German - Germany
+  { name: 'Deutsch', code: 'de-DE' },
+  // German - Austria
+  { name: 'Deutsch (Österreich)', code: 'de-AT' },
+  // German - Switzerland
+  { name: 'Deutsch (Schweiz)', code: 'de-CH' },
+  // Divehi/Maldivian - Maldives
+  { name: 'ދިވެހި', code: 'dv-MV' },
+  // Greek - Greece
+  { name: 'Ελληνικά', code: 'el-GR' },
+  // English - US
+  { name: 'English', code: 'en-US' },
```

Comment:
we have 2 en-US currently, probably remove this one and leave the `English (United States)` one so it's clear?